src/app.ts

```diff
@@ -1,6 +1,9 @@
 import {ProcessingManager} from "./processing-manager";
 import {Worker} from "./worker";
+import {TestRuns} from "./test-runs";
 
 
-ProcessingManager.run();
-Worker.run();
\ No newline at end of file
+//ProcessingManager.run();
+//Worker.run();
+
```

Comment:
**No, please don't!** Do not touch `app.ts` for benchmarking and testing... If you want to run a benchmark in `TestRuns`, add a line at the bottom of `TestRuns` which runs it for you when you run `test-runs.js` with Node.